cli/bin/grain.js

```diff
@@ -123,6 +138,10 @@ program
     "--elide-type-info",
     "don't include runtime type information used by toString/print"
   )
+  .profileOption(
+    "--release",
+    "produce WebAssembly modules suitable for production"
```

Comment:
Not sure how I feel about `suitable for production`. We don't want to just say `with the release profile`?

Reply:
```suggestion
    "compile using the release profile (production mode)"
```